raiden/transfer/channel.py

```diff
@@ -1891,6 +1894,29 @@ def handle_action_withdraw(
     return TransitionResult(channel_state, events)
 
 
+def handle_action_set_reveal_timeout(
+    channel_state: NettingChannelState, state_change: ActionChannelSetRevealTimeout
+) -> TransitionResult[NettingChannelState]:
+    events: List[Event] = list()
+
+    is_valid_reveal_timeout = (
+        state_change.reveal_timeout > 0
+        and channel_state.settle_timeout >= state_change.reveal_timeout * 2
```

Comment:
This is too strict.

Reply:
Discussed this internally. We agree that reveal timeout could be up to settle timeout or slightly smaller but since this exact condition is used in alot of places, we'll change that later on.